targets/lantiq-xrx200

```diff
@@ -1,3 +1,8 @@
+device('avm-fritz-box-7362-sl', 'avm_fritz7362sl', {
+        factory = false,
+        broken = true, -- Both LAN Ports work as LAN in Config and WAN in working mode
```

Comment:
See #1886 - this should qualify for non-broken then.

However, can you elaborate on the installation? Is it possible to add this specific model to fritz-tools / fritzfla.sh?